### PR TITLE
MAP-1872 fixing main branch

### PR DESCRIPTION
**Pull request by michalnawrockidj**
This is temporally fix to move forward with:
- Switching security alerts to Github actions
- Security update
The following action will be taken after that
 - Update hmpps-external-users-api to the newest version
 - Run all integration tests in specific order by Grade not by CircleCi
 - Reduce logs generated by Selenium webdriver
 - Review how useful is to have a screenshot when tests are failing 
  (Remove functionality or improve code by using TestWatcher)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
           - SERVER_PORT=9090
           - SPRING_PROFILES_ACTIVE=dev,local-postgres,auth-seed
           - SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:7432/auth-db
-      - image: quay.io/hmpps/hmpps-external-users-api:latest
+      - image: quay.io/hmpps/hmpps-external-users-api:2024-09-23.3630.2e6d15e
```
Comment (marcusaleman) [inline .circleci/config.yml:53]: this is a temporary image to downgrade de version as per discussion. right

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

**Review by jimbali — APPROVED** [review]
You are our knight in shining armour for fixing this, thanks 🙏

File: .circleci/config.yml (hunk 1 above)
Comment (marcusaleman) [reply .circleci/config.yml:53]: + we need to merge and  check if we need to merge fewer PR pendant still there.

**Review by marcusaleman — APPROVED** [review]
LGTM

**Pull request merged**